src/components/testimonialCard/testimonial-card.css

```diff
@@ -0,0 +1,56 @@
+.testimonial-container{
+  background-color: rgb(71, 79, 197);
+  height: 100%;
+  border-radius: 1rem;
+  position: relative;
+}
+
+.testimonial-container__purple{
```

Comment:
Check your usage of BEM naming conventions, e.g.
block__element--modifier
So, with `.testimonial-container__purple`, it should rather be: `.testimonial-container--purple`